vueless.config.ts

```diff
@@ -6,7 +6,9 @@ export default {
   ringOffset: 0,
   ringOffsetColorLight: "#ffffff", // white
   ringOffsetColorDark: "#111827", // gray-900
+  "rounding-sm": 4,
```

Comment:
use camelCase for config keys:
"rounding-sm" >>> roundingSm
"rounding-lg" >>> roundingLg

Need to add this new roundings into twMerge config also